### DISCUSSION TITLE
some icons are not visible in light mode on the dashboard page

### DISCUSSION THREAD

**Discussion by AbhijeetBiswal** [supabase/supabase#17622] [Contribute to Supabase]
some icons are not visible when we are in the light mode on the dashboard page. 
![invisible icons](https://github.com/supabase/supabase/assets/144774379/2e20a10a-8b9a-4049-9d69-812653fe482a)

**Comment by jailbreakerVC** [comment]
I'm gonna try working on this issue, please assign it to me if that's possible. Thanks
